maro/cli/inspector/cim_dashboard.py

```diff
@@ -0,0 +1,473 @@
+import math
+import os
+
+import numpy as np
+import pandas as pd
+
+import altair as alt
+import maro.cli.inspector.dashboard_helper as common_helper
+import streamlit as st
+from maro.cli.inspector.params import CIMItemOption
+from maro.cli.inspector.params import GlobalFilePaths as Gfiles
+from maro.cli.inspector.params import GlobalScenarios
+from maro.cli.inspector.visualization_choice import (CIMIntraViewChoice,
+                                                     PanelViewChoice)
+
+
+def start_cim_dashboard(source_path: str, epoch_num: int, prefix: str):
+    """Entrance of cim dashboard.
+
+    Args:
+        source_path (str): Data folder path.
+        epoch_num (int) : Number of data folders.
+        prefix (str): Prefix of data folders.
```

Comment:
generally, what will be the prefix? sth like "Port001", "Vessel001", or what?

Reply:
The prefix is ​​used to locate each snapshot folder. Prefix = "epoch_" currently.

Reply:
if the "prefix" is pre-defined in our logging system, why should we use it as a parameter?